### DISCUSSION TITLE
KDE without hardware acceleration?

### DISCUSSION THREAD

**Discussion by jsiddall** [Joshua-Riek/ubuntu-rockchip#399] [Q&A] [answered]
I see KDE Plasma is iffy with Wayland and the GPU drivers in their current state:

https://github.com/Joshua-Riek/ubuntu-rockchip/discussions/248

But what about without acceleration?  Can KDE be run stable with all the fancy new stuff disabled?

**Comment by jsiddall** [comment, marked as answer]
Update: no, about the only thing that was obvious to do was disable compositing, but that really didn't make anything work better.  KDE 5.27 (as discussed in another topic) does work significantly better, but still lots of issues regardless of whether using Wayland or X11.

**Reply by Eternalightwithin** [reply]
How is XFCE desktop? Or is Gnome the only one that does wayland good

**Reply by jsiddall** [reply]
Sorry, never tried XFCE.  I am living with the issues with KDE 5.27 but they are annoying.

**Reply by Joshua-Riek** [reply]
Gnome with Wayland is really the only stable and well preforming desktop.
